src/pages/roadmap.md

```diff
@@ -8,34 +8,46 @@ Timelines are estimates and subject to change. For information about delivered f
 
 ## Release cycle
 
-We typically follow this release cycle:
+Releases typically follow this cycle:
 
-- Alpha: Super early peek into the feature being built. Not ready for use in production apps.
-- Beta: Opportunity to test, validate, and provide feedback. Not ready for use in production apps.
-- Production: Ready for use in production apps.
+- Alpha: Super early peek into the feature being built. Not ready for use in production apps
+- Beta: Opportunity to test, validate, and provide feedback. Not ready for use in production apps
+- Production: Ready for use in production apps
 
-## Features roadmap
+## Messaging roadmap
+
+Roadmap for the messaging protocol and the SDKs and tools that empower developers to build with XMTP.
 
 ### 2023 Q3
 
+- XMTP v3 in Alpha  
+  - Forward secrecy, post-compromise security, and per-app installation inbox management implementations
+  - CLI tooling for E2E testing by internal team and curious developer partners
+  - Announcement of developer calls that enable the community to keep up with and contribute to XMTP v3’s progress
```

Comment:
```suggestion
```

Not a product roadmap item.

Reply:
Along these lines, for comms about the release, Esra'a, Rochelle, and I were wondering if these dev calls are still in the plans? When might they begin? Who will lead them?